android-tv/build.gradle

```diff
@@ -40,7 +40,7 @@ android {
 
 dependencies {
 
-    implementation 'androidx.core:core-ktx:1.5.31'
+    implementation 'androidx.core:core-ktx:1.3.0'
```

Comment:
Why are we downgrading?

Reply:
There was some build issue, so I used the sample-kotlin app version over here.

Reply:
Updated the version to 1.6.0